### PR TITLE
Autodesk APS fix naming and updated URLs

### PR DESCRIPTION
**Pull request by SuperDJ**
- Fixed naming;
- Added `region` as a config parameter;
- Fixed docs URL to match https://github.com/SocialiteProviders/website/pull/39;

Since the repository name is https://github.com/SocialiteProviders/AutodeskAPS, we might want to change the composer package name as well to something like `socialiteproviders/autodesk-aps`. The only 2 installs of the [package](https://packagist.org/packages/socialiteproviders/aps) at the moment are from me.

### CODE REVIEW

File: src/AutodeskAPS/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Please see the [Base Installation Guide](https://socialiteproviders.com/usage/),
   'client_id' => env('AUTODESK_APS_CLIENT_ID'),  
   'client_secret' => env('AUTODESK_APS_CLIENT_SECRET'),  
   'redirect' => env('AUTODESK_APS_REDIRECT_URI'),
+  'region' => env('AUTODESK_APS_REGION'), // Could add a default of EU or US
```
Comment (lucasmichot) [inline src/AutodeskAPS/README.md:21]: Where is it used?

Comment (SuperDJ) [reply src/AutodeskAPS/README.md:21]: @lucasmichot sorry for the late response. It isn't used in the provider right now. But for additional Autodesk API requests it's often required to specify a region.

**Comment by SuperDJ** [comment]
Marking this as draft as Autodesk will change authentication URLs https://aps.autodesk.com/blog/migration-guide-oauth2-v1-v2

File: src/AutodeskAPS/Provider.php [hunk 2]
```diff
@@ -57,22 +57,23 @@ protected function getUserByToken($token): array
     }
 
     /**
-     * https://forge.autodesk.com/en/docs/oauth/v2/reference/http/users-@me-GET/.
+     * https://aps.autodesk.com/en/docs/oauth/v2/reference/http/userinfo-GET/.
```
Comment (lucasmichot) [inline src/AutodeskAPS/Provider.php:60]:
```suggestion
     * @see https://aps.autodesk.com/en/docs/oauth/v2/reference/http/userinfo-GET/.
```

**Review by lucasmichot — APPROVED** [review]
Thanks @SuperDJ 👍 
GTM @atymic

**Pull request merged**